packages/node/src/indexer/store.service.ts

```diff
@@ -80,6 +95,7 @@ export class StoreService {
   ): Promise<void> {
     this.schema = schema;
     this.modelsRelations = modelsRelations;
+    this.historical = argv['experimental-historical'];
```

Comment:
We need further checks that the project was started with this enabled, it should not be possible to add this in after a project has started indexing.

There also needs to be handling for if the project is started with this flag then it is no longer enabled. I think we can throw an error in this case.

Reply:
done